src/Skylight.Protocol.X/Skylight.Protocol.WIN63-202111081545-75921380/packets.json

```diff
@@ -1552,13 +1552,13 @@
         },
         {
           "Mapping": "Field",
-          "Type": "List<Link>",
-          "Name": "Links"
+          "Type": "int",
+          "Name": "TrackingId"
         },
         {
           "Mapping": "Field",
-          "Type": "int",
-          "Name": "TrackingId"
+          "Type": "List<Link>",
+          "Name": "Links"
```

Comment:
Mistake? This should be reverted, this is wrong

Reply:
indeed a mistake